test/e2e/thanos_querier_test.go

```diff
@@ -21,66 +21,8 @@ import (
 	"time"
 
 	"github.com/openshift/cluster-monitoring-operator/test/e2e/framework"
-
-	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-func TestThanosQuerierTrustedCA(t *testing.T) {
```

Comment:
Should we wait for https://github.com/openshift/cluster-monitoring-operator/pull/2325/files?
/hold
unhold if not
/lgtm